temboardui/plugins/pgconf/templates/configuration.html

```diff
@@ -264,12 +263,11 @@ <h4 class="modal-title" id="reloadModalLabel">Are you sure you want to reload Po
               {% else %}
               <input data-toggle="popover" data-trigger="hover" data-placement="auto top" data-html="true" data-content="<table><tr><td>Type:</td><td><b>{{setting_row['vartype']}}</b></td></tr>{% if setting_row['unit'] %}<tr><td>Unit:</td><td><b>{{setting_row['unit']}}</b></td></tr>{% end %}{% if setting_row['vartype'] in ['integer', 'real'] %}<tr><td>Minimum:</td><td><b>{{setting_row['min_val']}}</b></td></tr><tr><td>Maximum:</td><td><b>{{setting_row['max_val']}}{% end %}</b></td></tr></table>" type="text" class="form-control input-sm" name="{{setting_row['name']}}" id="input{{setting_row['name']}}" placeholder="{{setting_row['name']}}" value="{%if setting_row['setting'] is not None%}{{setting_row['setting_raw']}}{%end%}">
               {% end %}
-            </td>
-            <td class="action-setting">
               {% if setting_row['setting_raw'] == setting_row['auto_val_raw'] and ((setting_row['file_val_raw'] and setting_row['setting_raw'] != setting_row['file_val_raw']) or (not setting_row['file_val_raw'] and setting_row['setting'] != setting_row['boot_val']))%}
-              <button type="button" id="buttonReset_{{setting_row['name']}}" data-toggle="popover" data-trigger="hover" data-placement="auto right" title="{{setting_row['name']}}" data-html="true" data-content="Reset to: '{% if not setting_row['file_val_raw'] %} {%if setting_row['boot_val']%}{{setting_row['boot_val']}}{%end%} {% else %} {%if setting_row['file_val_raw']%}{{setting_row['file_val_raw']}}{%end%} {% end %}'">
+              <a class="btn btn-link" id="buttonReset_{{setting_row['name']}}" data-toggle="popover" data-trigger="hover" data-placement="auto right" title="{{setting_row['name']}}" data-html="true" data-content="Reset to: '{% if not setting_row['file_val_raw'] %} {%if setting_row['boot_val']%}{{setting_row['boot_val']}}{%end%} {% else %} {%if setting_row['file_val_raw']%}{{setting_row['file_val_raw']}}{%end%} {% end %}'">
```

Comment:
Why `a` instead of `button` ? `button` looks more semantic (if this make sens anymore).